apps/system/js/app_install_manager.js

```diff
@@ -66,23 +68,13 @@
       this.notifContainer =
               document.getElementById('install-manager-notification-container');
       this.appInfos = {};
+      this.customizationApps = new Set();
```

Comment:
Sounds reasonable to me, though I'm not sure what you mean by migrating. Maybe Service.query('ftucustomization.contains', manifestURL).